src/test/java/cs2103/v15_1j/jimjim/JJStorageTest.java

```diff
@@ -0,0 +1,28 @@
+package cs2103.v15_1j.jimjim;
+
+import static org.junit.Assert.*;
+
+import java.io.FileNotFoundException;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class JJStorageTest {
+	private JJStorage storage;
+
+	@Before
+	public void setUp() throws Exception {
+		storage = new JJStorage("test.xml");
+	}
+	
+	@Test
+	public void testCreate() throws FileNotFoundException {
+		Task task1 = new Task("task1", LocalDateTime.now());
+		storage.create(task1);
+		
+		List<TaskEvent> list = storage.getAll();
+		assertEquals(1, list.size());
```

Comment:
I think this is not enough. I can pass this test case by just adding a random task to the list. The test should check that the name and datetime of the task is correct as well. (I think it's better to hard code some datetime using LocalDateTime.of instead of getting the current datetime)